NAMESPACE

```diff
@@ -130,11 +130,11 @@ S3method(melt, default)
 # and many packges on CRAN call dcast.data.table() and/or melt.data.table() directly. See #3082.
 export(melt.data.table, dcast.data.table)
 
-import(utils)
+importFrom(utils, capture.output, contrib.url, download.file, flush.console, getS3method, head, packageVersion, tail, untar, unzip)
 export(update_dev_pkg)
 S3method(tail, data.table)
 S3method(head, data.table)
-import(stats)
+importFrom(stats, as.formula, na.omit, setNames, terms, time)
```

Comment:
I think `time` is only used as NSE, we don't actually need to import `stats::time`:

https://github.com/Rdatatable/data.table/blob/815e054495d787f4a258fc3c6201a11f13e610e2/R/test.data.table.R#L197

Reply:
Instead, edit this line to define `time` as `NULL` as well:

https://github.com/Rdatatable/data.table/blob/815e054495d787f4a258fc3c6201a11f13e610e2/R/test.data.table.R#L194

That mimics what's done in the same file:

https://github.com/Rdatatable/data.table/blob/815e054495d787f4a258fc3c6201a11f13e610e2/R/test.data.table.R#L282

Reply:
Done :)